APITestingFramework/Features/Project/Post.feature

```diff
@@ -0,0 +1,22 @@
+Feature: Create a project
+
+    Scenario: Create a new project with valid parameters and valid authentication
+        Given the user is authenticated
+        And the request body contains the parameter "Content" with value "Project 1"
+        When the user sends a POST request to the "/projects" endpoint
```

Comment:
And the user fill the body with following data:
"""
{ 
  name: test
}

Reply:
in this part you should log and keep the project name and ID

Reply:
Done in the next commit